lxd/container_lxc.go

```diff
@@ -491,19 +491,23 @@ func parseRawIdmap(value string) ([]shared.IdmapEntry, error) {
 		switch entries[0] {
 		case "both":
 			entry.Isuid = true
-			ret.AddSafe(entry)
-			ret.AddSafe(shared.IdmapEntry{
-				Isgid:    true,
-				Hostid:   entry.Hostid,
-				Nsid:     entry.Nsid,
-				Maprange: entry.Maprange,
-			})
+			entry.Isgid = true
```

Comment:
Well, the rendering to LXC part was broken, but I fixed that in this pull request to.